models/sale.coffee

```diff
@@ -116,6 +116,10 @@ module.exports = class Sale extends Backbone.Model
   isRegistrationEnded: ->
     @isAuction() and moment().isAfter(@get 'registration_ends_at')
 
+  isLiveAuction: ->
+    @get('live_start_at') and moment(@get('live_start_at')).isBefore(moment(@get('end_at')))
+
+  # Is there a way to manually end/close out a live sale besides looking at end_at?
```

Comment:
@sweir27, @acjay  => I think we were talking about this yesterday? Just want to check on this piece since I think the end_at date might be irrelevant for live auctions.